src/App.tsx

```diff
@@ -55,6 +57,35 @@ loader.init().then((monaco) => {
   })
 })
 
+const getAppDataFromSearchParams = (): AppData => {
+  const urlParams = new URLSearchParams(window.location.search)
+  const encodedAppData = urlParams.get('appData')
+  if (!encodedAppData)
+    return {
+      schema: '',
+      values: [],
+    }
+
+  const decompressedAppData =
+    LZString.decompressFromEncodedURIComponent(encodedAppData)
+  const parsedAppData = JSON.parse(decompressedAppData)
```

Comment:
`parsed` should be omitted (or it should prefix a lot of declarations otherwise).

Reply:
done